services/121-service/src/registration/dto/duplicate.dto.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+// TODO: Discuss should this be name DuplicateReponseDto? (as in the guidelines) And that the api returns DuplicateReponseDto[]?
```

Comment:
An array of response seems weird. It's sort of one response which can have an arrays of duplicates